pyhf/tensor/mxnet_backend.py

```diff
@@ -293,7 +293,6 @@ def simple_broadcast(self, *args):
             ...   pyhf.tensorlib.astensor([1]),
             ...   pyhf.tensorlib.astensor([2, 2]),
             ...   pyhf.tensorlib.astensor([3, 3, 3]))
-            ...
```

Comment:
double-check that this actually needs to be removed. When you run in a python interpreter, there's always one more blank `...` in a block before you close it out. This is mapping that.

Reply:
Yeah, I mean, you usually see something like

```python
>>> def test():
...   print "1"
... 
```

Reply:
Yeah, exactly. Hence the need for the `...` in the `with` block in the TensorFlow backend.

Reply:
Well, I meant the trailing `...` here which you deleted, but ok. If the doctest is fine with it either way (e.g. just run pytest) then it's fine w/ me.

Reply:
We're in agreement in understanding about what is being talked about, even though we seem to both not be able to convey this to each other in text. Of course, if my REPL is different then the doc test for some reason it will go back.